modules/gapi/src/backends/ov/govbackend.cpp

```diff
@@ -235,7 +235,7 @@ struct OVUnit {
             compiled_model = cv::gapi::ov::wrap::getCore()
                 .compile_model(model, params.device, toOV(params.config));
         }
-        return {compiled_model};
+        return {compiled_model, params.nireq};
```

Comment:
`nireq` is not modified in `compile` or somewhere else and used as pipeline input. Why do you need it here?

Reply:
Well, it should be available from graph metadata in order to extract this value to create `RequestPool`. 
`OVCompiled` is the only structure which is  accessible from meta.

Reply:
Oh, got your point, let's just get it from `OVUnit`, thanks!

Reply:
Done